config.json

```diff
@@ -238,6 +238,18 @@
         "lists",
         "folding"
       ]
+    },
+    {
+      "slug": "allergies",
+      "uuid": "f964d022-1466-4c76-8877-0b0dda25578b",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 3,
+      "topics": [
```

Comment:
If it's not core, I believe you need to say which exercise unlocks it.

Reply:
I check that exercise on the Elm track and it is set as a core exercise, so I did it too.